ups/eupspkg.cfg.sh

```diff
@@ -1,5 +1,6 @@
 install()
 {
+    default_install
     mkdir -p "$PREFIX"
```

Comment:
I think you theoretically should be OK with removing this entire file.  But I'm not exactly sure what's going on with the installation.

Reply:
> I think you theoretically should be OK with removing this entire file. But I'm not exactly sure what's going on with the installation.

This is needed for copying the `yml` directory, which won't be done by default.